### PR TITLE
✨ add Week8

### PR DESCRIPTION
**Pull request by LouisaStrunk02**
Changes:

Trivia:
1. add index.html
2. changed styles.css

Homepage:
1. add index.html
3. add EMWinner.html
4. add WMWinner.html
5. add Player.html
6. add styles.css

### CODE REVIEW

File: week8/trivia/index.html [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+            <h3>In which country is it illegal to own only one guinea pig, as a lone guinea pig might get lonely?</h3>
+            <input type="text"></input>
+            <button id="check">Check Answer</button>
+
+            <p id="feedback2"></p>
+        </div>
+    </div>
+</body>
+
+</html>
```
Comment (stefanluth) [inline week8/trivia/index.html:70]: EOF

File: week8/trivia/styles.css [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+input[type="text"] {
+    line-height: 1.8;
+    width: 25%;
+}
+
+input[type="text"]:hover {
+    background-color: #b7ced2;
+    transition: color 2s ease-in-out, background-color 0.15s ease-in-out;
+}
```
Comment (stefanluth) [inline week8/trivia/styles.css:63]: EOF

File: week8/homepage/EMWinner.html [hunk 3]
```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
+
+<html lang="en">
+
+<head>
```
Comment (DevBastiS) [inline week8/homepage/EMWinner.html:5]: <head> Tag should be indeted.

HTML in general should have an indent of 2 spaces, plese check your editor configuration.

File: week8/homepage/EMWinner.html [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet"
+        integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/js/bootstrap.bundle.min.js"
+        integrity="sha384-MrcW6ZMFYlzcLA8Nl+NtUVF0sA7MsXsP1UyJoMp4YLEuNSfAP+JcXn/tWtIaxVXM"
+        crossorigin="anonymous"></script>
+    <link href="styles.css" rel="stylesheet">
+    <title>Homepage/example - Louisa </title>
```
Comment (DevBastiS) [inline week8/homepage/EMWinner.html:12]:
```suggestion
    <title>Homepage/example - Louisa</title>
```

File: week8/homepage/index.html [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+    <link href="name.js" rel="functionsheet">
+    <title>Homepage - Louisa </title>
+    <script>
+         document.addEventListener('DOMContentLoaded', function () {
+
+            document.querySelector("#submit").addEventListener("click", function () {
+                var input = document.getElementById("name").value;
+                document.querySelector("#feedback1").innerHTML = "Hello, " + input + ". Thank you for visiting my homepage :) My name is Louisa and i like playing soccer so this page is about women soccer.";
+                });
+         });
```
Comment (DevBastiS) [inline week8/homepage/index.html:22]: Move this script into a javascript file please.

File: week8/homepage/styles.css [hunk 6]
```diff
@@ -0,0 +1,172 @@
+header {
+    background-color: #11bfc896;
```
Comment (DevBastiS) [inline week8/homepage/styles.css:2]: CSS should have an indent of 2 spaces as well.

File: week8/trivia/index.html [hunk 7]
```diff
@@ -0,0 +1,70 @@
...
+                let input = document.querySelector("input");
+                if (input.value.toLowerCase() == "switzerland") {
+                    input.style.backgroundColor = "green";
+                    document.querySelector("#feedback2").innerHTML = "Correct";
+                } else {
+                    input.style.backgroundColor = "red";
+                    document.querySelector("#feedback2").innerHTML = "Incorrect";
+                }
+            });
+        });
```
Comment (DevBastiS) [inline week8/trivia/index.html:36]: Move to script file please.

File: week8/homepage/Player.html [hunk 8]
```diff
@@ -0,0 +1,62 @@
...
+    <link href="styles.css" rel="stylesheet">
+    <title>Homepage/example - Louisa </title>
+</head>
+
+<body>
+    <header>
+        <h1>My favorite players</h1>
+    </header>
+    <main>
+        <table id="Player" class="table table-hover">
```
Comment (DevBastiS) [inline week8/homepage/Player.html:20]: table-hover is not used in your css files.

**Review by DevBastiS — CHANGES_REQUESTED** [review]

File: week8/homepage/Player.html (hunk 8 above)
Comment (LouisaStrunk02) [reply week8/homepage/Player.html]: table-hover is a feature from Bootstrap: https://getbootstrap.com/docs/4.0/content/tables/

File: week8/homepage/Player.html [hunk 9]
```diff
@@ -0,0 +1,62 @@
...
+            <td>Midfielder</td>
+            <td>Olympique Lyon</td>
+            <td>German, Hungarian</td>
+          </tr>
+          <tr>
+            <td>Pernille Harder</td>
+            <td>Striker</td>
+            <td>FC Chelsea</td>
+            <td>Danish</td>
+          </tr>
```
Comment (stefanluth) [inline week8/homepage/Player.html:47]: I feel like there's one very famous and successful German player missing. :D

**Review by stefanluth — APPROVED** [review]
You could limit the name field to [a-zA-Z] but then again there are people like [Elon Musk](https://www.google.com/search?q=X+%C3%86+A-12+Musk). So I guess your version is future proof.

Screenshot 2022-09-06 at 08 47 47

**Comment by stefanluth** [comment]
:D

**Review by DevBastiS — APPROVED** [review]

**Review by stefanluth — APPROVED** [review]

**Pull request merged**